### DISCUSSION TITLE
django url in template with template in alpinejs

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3470] [1. Help]
I'm using AlpineJS in Django templates.

```python
path('view/<customer_id>', views.view, name='customers_view'),
```

This is the right approach `:href="{% url 'customers_view' customer_id='123' %}"`

But I am generating this in a template loop in alpineJS :

```
:href="'{% url 'customers_view' %}/' + cust_id"
```

How do I combine python in alpineJS to bypass this ?

```
Reverse for 'customers_view' with no arguments not found. 1 pattern(s) tried: ['customers/view/(?P<customer_id>[^/]+)\\Z']
```

**Comment by ekwoka** [comment]
You would need to provide a url that you can reverse to that does not require the customer id. or otherwise give a dummy one and replace it.
